content/resources/developer/dev-introduction.md

```diff
@@ -306,6 +320,12 @@ Versions must not be pinned in `setup.py`, however you may specify ranges:
 "requests>=2.19.1,<3.0.0"
 ```
 
+or preferably depend on [compatible releases](https://www.python.org/dev/peps/pep-0440/#compatible-release):
```

Comment:
mention that same syntax applies to pipenv install?

Reply:
Sure thing!